python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -218,3 +218,61 @@ def __new__(
             error=check.opt_inst_param(error, "error", SerializableErrorInfo),
             run_status=check.opt_inst_param(run_status, "run_status", DagsterRunStatus),
         )
+
+
+@experimental
+class SensorResult(
+    NamedTuple(
+        "_SensorResult",
+        [
+            ("run_requests", Optional[Sequence[RunRequest]]),
+            ("skip_reason", Optional[SkipReason]),
+            ("pipeline_run_reactions", Optional[Sequence[PipelineRunReaction]]),
+            ("cursor", Optional[str]),
+        ],
+    )
+):
+    """The result of a sensor evaluation.
+
+    Attributes:
+        run_requests (Optional[Sequence[RunRequest]]): A list
+            of run requests to be executed.
+        skip_reason (Optional[SkipReason]): A skip message indicating why sensor evaluation was skipped.
+        cursor (Optional[str]): The cursor value for this sensor, which will be provided on the
+            context for the next sensor evaluation.
+        pipeline_run_reactions (Optional[PipelineRunReaction]): A list of pipeline run reactions to be evaluated.
+    """
+
+    def __new__(
+        cls,
+        run_requests: Optional[Sequence[RunRequest]] = None,
+        skip_reason: Optional[SkipReason] = None,
+        pipeline_run_reactions: Optional[Sequence[PipelineRunReaction]] = None,
+        cursor: Optional[str] = None,
+    ):
+        if not run_requests and not skip_reason and not pipeline_run_reactions:
+            check.failed(
+                "Must provide at least one of run_requests, skip_reason, or pipeline_run_reactions"
```

Comment:
Any downside to removing this check? Sometimes nothing happens inside your sensor but you also don't have a skip reason. This would force people to include an if statement that they otherwise wouldn't need.

Reply:
ah yeah makes sense, looks like we automatically assign a skip message when one isn't provided anyway